%% file `template_en.tex‘, modern cv
%% Copyright 2006-1008 Xavier Danaux (xdanaux@gmail.com).
%
% This work may be distributed and/or modified under the
% conditions of the LaTeX Project Public License version 1.3c,
% available at http://www.latex-project.org/lppl/.


\documentclass[11pt,a4paper]{moderncv}

% moderncv themes
\moderncvstyle{classic} % style options are 'casual' (default), 'classic', 'banking', 'oldstyle' and 'fancy'
\moderncvcolor{blue} % color options 'black', 'blue' (default), 'burgundy', 'green', 'grey', 'orange', 'purple' and 'red'

% character encoding
\usepackage[utf8]{inputenc}                   % replace by the encoding you are using
\usepackage{pslatex}
\usepackage{textcomp}

% font loading
% for luatex and xetex, do not use inputenc and fontenc
% see https://tex.stackexchange.com/a/496643
\ifxetexorluatex
  \usepackage{fontspec}
  \usepackage{unicode-math}
  \defaultfontfeatures{Ligatures=TeX}
  \setmainfont{Latin Modern Roman}
  \setsansfont{Latin Modern Sans}
  \setmonofont{Latin Modern Mono}
  \setmathfont{Latin Modern Math}
\else
  \usepackage[utf8]{inputenc}
  \usepackage[T1]{fontenc}
  \usepackage{lmodern}
\fi

% document language
\usepackage[english]{babel}  % FIXME: using spanish breaks moderncv

% adjust the page margins<
\usepackage[scale=0.75]{geometry}
%\setlength{\hintscolumnwidth}{3cm}						% if you want to change the width of the column with the dates
%\AtBeginDocument{\setlength{\maketitlenamewidth}{6cm}}  % only for the classic theme, if you want to change the width of your name placeholder (to leave more space for your address details
\AtBeginDocument{\recomputelengths}                     % required when changes are made to page layout lengths
\makeatletter
\def\url@leostyle{%
  \@ifundefined{selectfont}{\def\UrlFont{\sf}}{\def\UrlFont{\small\rmfamily}}}
\makeatother

%% Now actually use the newly defined style.
\urlstyle{leo}


% personal data
\firstname{Jitao David}
\familyname{Zhang}
% \title{\fontsize{12}{14}\selectfont A Computational Biologist in Drug Discovery}               % optional, remove the line if not wanted
% \address{F. Hoffmann-La Roche AG}{4070 Basel, Switzerland}    % optional, remove the line if not wanted
% \address{Department of Mathematics and Computer Sciences, University Basel}{4070 Basel, Switzerland}    % optional, remove the line if not wanted
\email{jitao\_david.zhang@roche.com}
\homepage{jdzhang.me}

\renewcommand*{\bibliographyitemlabel}{[\arabic{enumiv}]}

%\nopagenumbers{}                             % uncomment to suppress automatic page numbering for CVs longer than one page


%----------------------------------------------------------------------------------
%            content
%----------------------------------------------------------------------------------
\begin{document}
\renewcommand*{\namefont}{\fontsize{32}{32}\sffamily\mdseries\upshape} 
%\renewcommand{\namefont}{\fontsize{30}{32}\sffamily\mdseries\upshape}
\maketitle

\section{Key facts}

\cvlistitem{Computational biologist with 10+ years' industrial research experience}
\cvlistitem{Key contributions to drug candidates in development and marketed products}
\cvlistitem{Inventing and implementing proprietary technologies and workflows}
\cvlistitem{Establishing a new research group on computational toxicology and safety}
\cvlistitem{Author of patents, open-source software, and publications (\textit{h}-index 29)}
\cvlistitem{Teaching two courses and supervising postdocs, students, and apprentices}

\section{Research and teaching}

\cventry{2011--\hspace{4.25ex}}{Expert Computational Biologist}{F. Hoffmann-La
Roche AG}{Basel, Switzerland}{}{I develop algorithms and software to mine,
interpret, model and integrate heterogeneous data, and apply mathematical and
computational tools to support preclinical drug discovery projects. I am
responsible for target assessment and validation, multiscale modeling and causal
inference of drug mechanism and safety, and MoA characterisation and de-risking
of drug candidates. With colleagues I co-develop novel platforms and resources
to support drug discovery, for instance
the~\href{https://en.wikipedia.org/wiki/Molecular_phenotyping}{molecular
phenotyping} platform, the Pathway-Annotated Chemical Ensemble (PACE) library,
and new methods for computational toxicology and safety assessment.}

\cventry{2018--\hspace{4.25ex}}{Lecturer}{Department of Mathematics and Computer Science, University of Basel}{}{}{Teaching the lecture series~\textit{Introduction to \textbf{A}pplied \textbf{M}athematics and \textbf{I}nformatics In \textbf{D}rug \textbf{D}iscovery} (\url{http://AMIDD.ch}) in autumn semesters, and~\textit{\textbf{M}athematical and \textbf{C}omputational \textbf{B}iology in \textbf{D}rug \textbf{D}iscovery} (\url{http://MCBDD.ch}) in spring semesters.}


\section{Training, mentoring, and academic commitment}
\cvitem{2022--\hspace{4.25ex}}{Co-organizer of Roche PMDA (Predictive Modeling and Data Analytics) Summer School for PhD students.}
\cvitem{2021--\hspace{4.25ex}}{Certified \textit{Lehrmeister} for informatics vocational apprenticeship.}
\cvitem{2020--\hspace{4.25ex}}{Certified Software and Data Carpentry instructor, hosting company internal carpentries.}
\cvitem{2018--\hspace{4.25ex}}{Participation in the lecture
series~\textit{From Novel Targets To Novel Therapeutic Modalities}, Master
Programme Drug Sciences, University of Basel.}
\cvitem{2016--\hspace{4.25ex}}{Supervising
master students in bioinformatics and computational biology, and co-supervise
PostDocs in collaboration with academic and industrial collaboration partners.}
\cvitem{2012--\hspace{4.25ex}}{Reviewing research proposals for funding agencies
and manuscripts for journals including \emph{Bioinformatics}, \emph{PLOS Comp.
Biol.}, \emph{NAR Genom. and Bioinform.}, \textit{etc.}}
\cvitem{2012--\hspace{4.25ex}}{Leading courses, seminars and
workshops about programming, bioinformatics, and computational biology in drug
discovery.}

\section{Education}
\cventry{2008--2011}{Dr.rer.nat. Bioinformatics}{German Cancer Research Center/ Universit\"at Heidelberg}{}{}{Computational and statistical approaches to study gene networks, supervised by Dr. Stefan Wiemann}
%I have also been working with image analysis of high-content microscopy.}
\cventry{2007-2007}{Marie-Curie Fellow}{Huber Group, European Institute of Bioinformatics}{}{Cambridge, UK}{}
\cventry{2006--2008}{M.Sc. Bioinformatics}{Universit\"at Heidelberg}{Germany}{}{}
\cventry{2002--2006}{B.Sc. Biology, hon.}{Peking University}{Beijing, China}{}{}  % arguments 3 to 6 are optional
% \cventry{1999--2002}{High-school diplom}{Tianjin Nr.1 High School}{Tianjin, China}{}{}  % arguments 3 to 6 are optional
%% B.Sc.: Mit Auszeichnung, Ranking top 5\%. Nebenfach: Germanistik, Ranking 1/30
%\section{Promotion}
%\cvline{Thema}{\emph{}}
%\cvline{Betreuer}{PD Dr. Stefan Wiemann, Abteilung Molekulare Genomanalyse, DKFZ}
%\cvline{Thesis Advisory Commitee}{Dr. Wolfgang Huber, Dr. Rainer K\"onig}
%\cvline{Kurzbeschreibung}{\small}

% Publications from a BibTeX file
\nocite{*}
\renewcommand\refname{Selected peer-reviewed publications}
\bibliographystyle{plainyr-rev}
\bibliography{mypubs-selected}       % 'publications' is the name of a BibTeX file
\cvitem{}{$\to$ See the full, manually curated list at \href{http://goo.gl/CoeJu7}{Google Scholar} (\url{http://goo.gl/CoeJu7}).}

\section{Selected invited talks}
\cvitem{2021}{\textit{Optimization of the TeraTox assay for preclinical teratogenicity assessment}, co-presentation with Manuela Jaklin, OpenTox Virtual Conference, 2021}
\cvitem{2019}{\textit{Bioinformatics and exploratory data analysis in drug discovery: an industrial perspective}, ISMB/ECCB, 2019, Basel, Switzerland}
\cvitem{2018}{\textit{Mathematics in drug discovery: a practitioner’s view}, Perlen-Kolloquium, University of Basel, Basel, Switzerland}

\section{Selected open-source software}
\cvitem{BESCA}{Single-cell omics data analysis pipeline, built together with BEDA colleagues.}
\cvitem{BioQC}{Detecting tissue heterogeneity in high-throughput expression
data.}
\cvitem{KEGGgraph}{Data mining and network analysis
  of biological pathways as graphs in R and Bioconductor.}

\section{Supervised postdocs}

\cvitem{Davide Bassani}{2023- (co-supervision with Dr. Neil Parrott and Dr. Nenad Manevski), computational approaches to preclinical pharmacokinetic property prediction}
\cvitem{Milad Adibi}{2020-2022 (co-supervision with Dr. Ekaterina Breous-Nystrom), multiscale modeling of drug-induced liver toxicity, currently senior computational biologist at University of Zurich}
\cvitem{Simon Gutbier}{2018-2020 (co-supervision with Dr. Christoph Patsch and Dr. Markus Britschgi), immune pathway characterization with tool-compound screening for Parkinson's Disease, currently principal scientist at Roche}

\section{Supervised master and PhD students}
\cvitem{Alexandra Valeanu}{2023, internship on inferring causal kinases with phosphoproteomics data}
\cvitem{Simon Crouzet}{2022, internship on predicting protein-ligand interaction with graph neural networks, currently PhD student in Dal Peraro's group at EPFL}
\cvitem{Anja Lieberherr}{2022-2023 (co-supervision with Prof. Niko Beerenwinkel), master thesis on applying graph neural networks in drug discovery, currently junior technology consultant at BearingPoint, Zürich}
\cvitem{Sarah Leonardo Morillo}{2021-2022, internship on proteomics data and protein-half life, currently PhD student in Erik van Nimwegen's group at Biozentrum, University of Basel}
\cvitem{Andreea Ciuprina}{2020 (co-supervision with Prof. Niko Beerenwinkel), internship and master thesis on computational inference of immune-cell contribution to drug-induced liver toxicity, currently data engineer and scientist at Endress+Hauser Flowtec}
\cvitem{Rudolf Biczok}{2018-2019 (co-supervision with Prof. Alexandros Stamatakis), internship and master thesis on building a gene-expression database and evaluating gene-set comparison metrics, currently full-stack Java architect and Azure cloud specialist at Bank for International Settlements (BIS)}
\cvitem{Moaraj Hasan}{2017-2018, internship on automating gene-expression data analysis}
\cvitem{Tao Fang}{2017-2018 (co-supervision with Prof. Mark Robinson), internship and master thesis on histopathology prediction with deep neural networks, currently postdoctoral fellow in von Mering group at University of Zurich}
\cvitem{Gregor Sturm}{2016-2017, master intern working on gene signatures and the BioQC software, currently Scientist Clinical Bioinformatics, Boehringer Ingelheim}

\section{Trained vocational apprentices in computer sciences}
\cvitem{Yannik Benndorf}{2022-2023, currently bachelor of computer sciences in TU München}
\cvitem{Paul Geser}{2022-2023, currently local IT manager volunteer at MAF Madagascar}
\cvitem{Leonardo Seminatore}{2021-2022, currently bachelor of science in geomatics, FHNW}

%\section{Talks}
%\section{Vortr\"age}
%\cventry{November 2008}{RpsiXML: Bridging PSI-MI XML standard with statistical and computational environment of R}{Workshop on the Development of Standards-compliant tools for molecular interaction data management, Cambridge, UK}{}{}{}

%\section{Poster and publications without review (selected)}
%\cventry{November 2010}{Catching the dynamics of RNAi phenotypes
%  during a human kinome screen}{System Genomics 2010, Heidelberg,
%  Germany}{\textbf{JD. Zhang}, Udo Eichenlaub, \textit{et al.}}{}{}
%\cventry{June 2009}{A novel large-scale screen to identify modulators
%  of miR-21}{MicroRNA and Cancer Meeting 2009, Colorado,
%  USA}{\textbf{JD. Zhang}, I. Keklikoglou, \textit{et al.}}{}{}
%\cventry{February 2009}{Analyzing the Effect of Gene Knock-Down in
%  Real-Time Using Roche's xCELLigence System}{Biochemica
%  2009/02}{\textbf{JD. Zhang}, A. Duda, \textit{et al.}}{}{}
%\cventry{February 2009}{qPCR Identification of Genes Involved in Apoptosis and Cell Cycle Regulation}{Biochemica 2009/02}{\textbf{JD. Zhang}, U. Ernst, A. Irsigler, S. Wiemann and U. Tschulena}{}{}
%\cventry{Mai 2008}{DNAdeconvolutor: software for DNA histogram analysis in R and Bioconductor}{Systems Genomics Meeting 2008, Heidelberg}{\textbf{JD. Zhang}, S. Bechtel, S. Wiemann and A. Poustka}{}{}
%\cventry{August 2007}{Protein Interaction Pool (PIP) -- A comprehensive database and interface to protein interaction data}{ISCB 2007, Wien}{J. Kunert, T. Beissbarth, \textbf{JD. Zhang}, A. Poustka, S. Wiemann and A. Mehrle}{}{}

\renewcommand{\listitemsymbol}{-} % change the symbol for lists
%\section{Current projects}
%\section{Aktuelle Projekte}
%\cvlistitem{Impact of microRNA regulation on biological pathways}
%\cvlistitem{Cell-based screening to identify regulators of human microRNA miR-21 (Kooperation mit Dr. Ulrich Tschulena)}
%\cvlistitem{New cell-based screening with multi-demensional output to identify novel regulators of ERBB signaling system (Kooperation mit Dr. \"Ozg\"ur Sahin)}



\vspace*{-1em}
\section{Personal information}
\cvitem{Date of Birth}{Sept. 28$^{th}$, 1983 in Tianjin, China}
\cvitem{Marital status}{Married, father of two daughters}
\cvitem{Languages}{English, German, Chinese}
\end{document}

%\renewcommand{\listitemsymbol}{-} % change the symbol for lists

%\section{Extra 1}
%\cvlistitem{Item 1}
%\cvlistitem{Item 2}
%\cvlistitem[+]{Item 3}            % optional other symbol

%\section{Extra 2}
%\cvlistdoubleitem[\Neutral]{Item 1}{Item 4}
%\cvlistdoubleitem[\Neutral]{Item 2}{Item 5}
%\cvlistdoubleitem[\Neutral]{Item 3}{}





%% end of file `template_en.tex'.
